scaffolder-templates/build/.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+stages:
+  - wait
+  - update
+
+wait_for_value:
+  tags:
+    - shared-podman
+  stage: wait
+  image: ubuntu:latest
+  script: |
+    sleep 5
+
+update_pipelinerun:
+  tags:
+    - shared-podman # Change to whatever tag your Gitlab runner is set to
```

Comment:
either make it configurable via the UI or at least add something to the project README (if not done already), like it's done for the gitops setup?